distributed/client.py

```diff
@@ -334,7 +334,7 @@ def _repr_html_(self):
             except AttributeError:
                 typ = str(self.type)
             text += '<font color="gray">type: </font>%s, ' % typ
-        text += '<font color="gray">key: </font>%s' % html_escape(self.key)
+        text += '<font color="gray">key: </font>%s' % html_escape(str(self.key))
```

Comment:
Not to be pedantic, but do you think this should be `str` or `repr`?

Reply:
Pedantry is welcome.  When would this matter?

In practice keys are only of type str, bytes, or tuple, but this isn't enforced so a sane default here would be good.

Reply:
Not in any current cases that I can think of.

There is a little bit of subtlety between `repr` and `str`. The former typically returns something that can be evaluated to get something equivalent. Also `repr` and `str` behave slightly differently with `bytes` vs. `str` on Python 2 or Python 3. 

<details>
<summary>Python 2</summary>

```python
In [1]: repr(b"a")
Out[1]: "'a'"

In [2]: repr(u"a")
Out[2]: "u'a'"

In [3]: str(b"a")
Out[3]: 'a'

In [4]: str(u"a")
Out[4]: 'a'
```

</details>

<details>
<summary>Python 3</summary>

```python
In [1]: repr(b"a")
Out[1]: "b'a'"

In [2]: repr(u"a")
Out[2]: "'a'"

In [3]: str(b"a")
Out[3]: "b'a'"

In [4]: str(u"a")
Out[4]: 'a'
```

</details>

Reply:
`str` it is